src/makeCalculator.js

```diff
@@ -33,11 +33,56 @@
  *
  * calculator.result === -28
  *
- *
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const add = num => {
+    result += num;
+
+    return makeCalculator();
+  };
+
+  const subtract = num => {
+    result -= num;
+
+    return makeCalculator();
+  };
+
+  const multiply = num => {
+    result *= num;
+
+    return makeCalculator();
+  };
+
+  const divide = num => {
+    result /= num;
+
+    return makeCalculator();
+  };
+
+  const reset = () => {
+    result = 0;
+
+    return makeCalculator();
+  };
```

Comment:
Return of 'makeCalculator' in the 'reset' method is incorrect. It creates a new calculator each time. You should return 'this' to enable chaining.